executor/wasm-host/src/host.rs

```diff
@@ -466,10 +466,10 @@ pub fn casper_create<S: GlobalStateReader + 'static, E: Executor + 'static>(
         caller.bytecode()
     };
 
-    let value: u128 = {
-        let mut value_bytes = [0u8; 16];
+    let value = {
+        let mut value_bytes = [0u8; 8];
         caller.memory_read_into(value_ptr, &mut value_bytes)?;
-        u128::from_le_bytes(value_bytes)
+        u64::from_le_bytes(value_bytes)
```

Comment:
No need for pointer manipulation. It'd be simpler if u64 is passed by value in this case